lib/game/configureGamePass.js

```diff
@@ -0,0 +1,100 @@
+// Includes
+const http = require('noblox.js/lib/util/http').func
+const getGeneralToken = require('noblox.js/lib/util/getGeneralToken').func
+
+// Args
+exports.required = ['id', 'name']
```

Comment:
'id' should be 'gamepassId' (consistency)

Reply:
Added. Opted for `gamePassId` in favor of `gamepassId` to be consistent with Roblox's "game-pass" naming scheme.